components/ScheduleSection.js

```diff
@@ -0,0 +1,240 @@
+        <SessionRow time='11:15 - 12:00'>
+          <SessionItem
+            topic='Live coding / mob programming session'
+            speakerName='Thai Pangsakulyanont'
+            speakerTitle='(Taskworld)'
+          />
+          <SessionItem
+            topic='Behavior Driven Development in React Native'
+            speakerName='Tanakorn Numrubporn'
+            speakerTitle='(CTO at Planforfit)'
+          />
+        </SessionRow>
+        <BreakRow time='Lunch Break' />
+        <ActivityRow time='13:00 - 14:10'>Lightning Talks</ActivityRow>
+        <SessionRow time='14:15 - 15:00'>
+          <SessionItem
+            topic='Switching from Mobile to React Web Developer'
+            speakerName='Chinnawat Panmongkol'
+            speakerTitle='(Wongnai)'
+          />
+          <SessionItem
+            topic='Styled Components: Change your CSS styling mindset'
+            speakerName='Yanin Tuamsuk'
+            speakerTitle='(Front-end Engineer at Jitta)'
+          />
+        </SessionRow>
+        <BreakRow time='Break' />
+        <SessionRow time='15:15 - 16:00'>
+          <SessionItem
+            topic='Angualar to React without breakingthings'
+            speakerName='Jirat Kijlerdpornpailoj'
+            speakerTitle='(SPRINT3r / Gucode)'
+          />
+          <SessionItem
+            topic='Testable React: Patterns and comparison'
+            speakerName='Chakrit Likitkhajorn'
+            speakerTitle='(Lead Developer at Taskworld)'
+          />
+        </SessionRow>
+        <SessionRow time='16:00 - 16:45'>
+          <SessionItem
+            topic='Workshop with Next.js'
+            speakerName='Nati Namvong'
+            speakerTitle='(CTO at Sellsuki)'
+          />
+          <SessionItem
+            topic='Lessons Learned from using Next.js in Production'
+            speakerName='Panjamapong Sermsawatsri'
+            speakerTitle='(CTO at TakeMeTour)'
+          />
+        </SessionRow>
+      </div>
+      <style jsx>{`
+        .schedule {
+          text-align: center;
+        }
+        .container {
+          max-width: 1060px;
+          width: 100%;
+          margin: 0 auto;
+          text-align: left;
+        }
+        @media (max-width: 768px){
+          .container {
+            max-width: 750px;
+            text-align: center;
+          }
+        }
+      `}</style>
+    </div>
+  )
+}
+
+const getSpeakerAvatar = (name) => {
+  return `static/speakers/${name.replace(' ', '')}.jpg`
+}
+
+const ScheduleRow = ({ children }) => (
+  <div className='row _margin-bottom-30'>
+    {children}
+    <style jsx>{`
+      .row {
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left:7.5px;
+        padding-right:7.5px;
+      }
+      ._margin-bottom-30 {
+        margin-bottom: 30px;
+      }
+    `}</style>
+  </div>
+)
+
+const ScheduleColumn = ({ children, size }) => {
+  return (
+    <div className='column' data-size={size}>
```

Comment:
Let’s add a prop `alignSelf`, to allow setting column alignment.

```js
     <div className='column' data-size={size} style={{ alignSelf }}>
```

This will fix few problems that I will elaborate in the next comments: